File: Modul-1/Soal-10/PRAK110-2410817320001-NazlaSalsabila.py
import math

# Diketahui
alas = 5
tinggi = 12

# Menghitung sisi miring (sisi C) menggunakan Teorema Pythagoras
sisi_miring = math.sqrt(alas**2 + tinggi**2)

# Menghitung keliling
keliling = alas + tinggi + sisi_miring

# Menghitung luas
luas = 0.5 * alas * tinggi

# Menampilkan output
print("Diketahui :")
print("Alas = %d cm" % alas)
print("Tinggi = %d cm" % tinggi)
print()
print("Jawab :")
print("Sisi A = %d cm" % alas)
print("Sisi B = %d cm" % tinggi)
print("Sisi C = %.0f cm" % sisi_miring)
print("Keliling = %.0f cm" % keliling)
print("Luas = %.0f cm" % luas)

File: Modul-1/Soal-10/PRAK110-2410817320001-NazlaSalsabila.c
#include <stdio.h>
#include <math.h>

int main() {
    // Diketahui
    int alas = 5;
    int tinggi = 12;

    // Menghitung sisi miring (sisi C) menggunakan Teorema Pythagoras
    double sisi_miring = sqrt(pow(alas, 2) + pow(tinggi, 2));

    // Menghitung keliling
    double keliling = alas + tinggi + sisi_miring;

    // Menghitung luas
    double luas = 0.5 * alas * tinggi;

    // Menampilkan output
    printf("Diketahui :\n");
    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n", tinggi);
    printf("\n");
    printf("Jawab :\n");
    printf("Sisi A = %d cm\n", alas);
    printf("Sisi B = %d cm\n", tinggi);
    printf("Sisi C = %.0f cm\n", sisi_miring);
    printf("Keliling = %.0f cm\n", keliling);
    printf("Luas = %.0f cm\n", luas);

    return 0;
}
